<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadWindow</class>
 <widget class="QWidget" name="DownloadWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>yt-dlp</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="audio_vlayout">
    <item>
     <widget class="QPushButton" name="audio_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Audio</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="audio_options_vlayout">
      <item>
       <layout class="QVBoxLayout" name="audio_ext_vlayout">
        <item>
         <widget class="QLabel" name="audio_ext_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Audio Extension:&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="audio_extension_hlayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="audio_codec_vlayout">
        <item>
         <widget class="QLabel" name="audio_codec_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Audio Codec:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="audio_codec_hlayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="top_hlayout">
    <item>
     <widget class="QLineEdit" name="input_url_field">
      <property name="placeholderText">
       <string>Enter the video URL here</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="video_vlayout">
    <item>
     <widget class="QPushButton" name="video_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="video_options_vlayout">
      <item>
       <layout class="QVBoxLayout" name="resolution_vlayout">
        <item>
         <widget class="QLabel" name="resolution_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Resolution:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="resolution_hlayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="fps_vlayout">
        <item>
         <widget class="QLabel" name="fps_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;FPS:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fps_hlayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="video_extension_vlayout">
        <item>
         <widget class="QLabel" name="video_extension_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Video Extension:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="video_extension_hlayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="video_codec_vlayout_2">
        <item>
         <widget class="QLabel" name="video_codec_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Video Codec:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="video_codec_hlayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="video_codec_vlayout">
        <item>
         <widget class="QLabel" name="vaudio_codec_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Audio Codec:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="vaudio_codec_hlayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="download_hlayout">
    <item>
     <widget class="QTextEdit" name="output_download_options">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="download_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
